Bug: build failure

### Current Behaviour

/home/azero/azerothcore-wotlk/modules/mod-individual-xp/src/individual_xp.cpp:66:10: error: ‘void IndividualXP::OnLogin(Player*)’ marked ‘override’, but does not override
   66 |     void OnLogin(Player* player) override
      |          ^~~~~~~
compilation terminated due to -Wfatal-errors.
make[2]: *** [modules/CMakeFiles/modules.dir/build.make:636：modules/CMakeFiles/modules.dir/mod-individual-xp/src/individual_xp.cpp.o] error 1


### Expected Behaviour

build success

### Steps to reproduce the problem

git clone 
cmake 
g++ make



### Extra Notes

_No response_

### AC rev. hash/commit

last version

### Operating system

ubuntu24.04

### Custom changes or Modules

_No response_

--- a/src/individual_xp.cpp
+++ b/src/individual_xp.cpp
@@ -63,7 +63,7 @@ class IndividualXP : public PlayerScript
 public:
     IndividualXP() : PlayerScript("IndividualXP") {}
 
-    void OnLogin(Player* player) override
+    void OnPlayerLogin(Player* player) override
     {
         QueryResult result = CharacterDatabase.Query("SELECT `XPRate` FROM `individualxp` WHERE `CharacterGUID`='{}'", player->GetGUID().GetCounter());
 
@@ -101,7 +101,7 @@ public:
         }
     }
 
-    void OnLogout(Player* player) override
+    void OnPlayerLogout(Player* player) override
     {
         if (PlayerXpRate* data = player->CustomData.Get<PlayerXpRate>("IndividualXP"))
         {
@@ -109,7 +109,7 @@ public:
         }
     }
 
-    void OnGiveXP(Player* player, uint32& amount, Unit* /*victim*/, uint8 /*xpSource*/) override
+    void OnPlayerGiveXP(Player* player, uint32& amount, Unit* /*victim*/, uint8 /*xpSource*/) override
     {
         if (individualXp.Enabled)
         {

